web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -70,46 +70,57 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
       <div className="d-flex togglers-wrapper">
         {stateColorTuples.map(([state, color]) => {
           return (
-            <Checkbox wrapperStyles={{ marginRight: 10 }} defaultChecked id={`${state}-toggler`} onClick={toggle(state)}>
+            <Checkbox
+              key={color}
```

Comment:
Might be better to key on `state`, just in case colors are ever not unique in the future.

Reply:
yes, ```state``` would be better, thanks!